### PR TITLE
Wdfn 516 - Reorder and refine the submenus

### PR DESCRIPTION
**Pull request by mbucknell**
Before making a pull request
----------------------------

- [X] Make sure all tests run
- [X] Update the changelog appropriately

Title
-----------
Wdfn 516 - Reorder and refine the submenus

After making a pull request
---------------------------
- [ ] If appropriate, put the link to the PR in the JIRA ticket
- [ ] Assign someone to review unless the change is trivial

### CODE REVIEW

File: wdfn-server/waterdata/templates/partials/header.html [hunk 1]
```diff
@@ -88,35 +117,20 @@
                             </a>
                         </li>
                         <li class="usa-nav__submenu-item">
-                            <a href="https://www.usgs.gov/special-topic/water-science-school"
-                               rel="noopener" target="_blank"
-                               ga-on="click" ga-event-category="navigation" ga-event-action="WaterScienceSchoolNav">
-                                How Data are Collected
-                            </a>
+                            <a href="https://waterservices.usgs.gov/">USGS Water Data APIs</a>
```
Comment (abriggs-usgs) [inline wdfn-server/waterdata/templates/partials/header.html:120]: Does this one need the target and other attributes?

File: wdfn-server/waterdata/templates/partials/header.html [hunk 2]
```diff
@@ -88,35 +117,20 @@
...
-                        </li>
-                        <li class="usa-nav__submenu-item">
-                            <a href="https://labs.waterdata.usgs.gov/index.html" rel="noopener">Experimental Water Data tools</a>
+                            <a href="https://www.usgs.gov/mission-areas/water-resources/science/water-data-visualizations?qt-science_center_objects=0#qt-science_center_objects" rel="noopener">
+                                Water Data Visualizations
+                            </a>
                         </li>
                         <li class="usa-nav__submenu-item">
-                            <a href="https://www.instagram.com/usgs_streamgages/" rel="noopener">Streamgages of Instagram</a>
+                            <a href="https://dashboard.waterdata.usgs.gov/app/nwd/">National Water Dashboard</a>
```
Comment (abriggs-usgs) [inline wdfn-server/waterdata/templates/partials/header.html:131]: Should these open in new tabs?

**Review by abriggs-usgs — APPROVED** [review]
Looks good. I just had some questions about attributes.

File: wdfn-server/waterdata/templates/partials/header.html (hunk 1 above)
Comment (mbucknell) [reply wdfn-server/waterdata/templates/partials/header.html:120]: It is our product so I am assuming not and we didn't before. This one was just picked up and moved.

File: wdfn-server/waterdata/templates/partials/header.html (hunk 2 above)
Comment (mbucknell) [reply wdfn-server/waterdata/templates/partials/header.html:131]: It doesn't in the link in the USGS header so I had it behave the same here. It is really part of WDFN.

**Pull request merged**